.gitlab-ci.yml

```
@@ -734,7 +734,8 @@ deploy-kubernetes-alerting-rules:
     refs:
       - master
     changes:
-      - "${RULES}"
+      - .gitlab-ci.yml
+      - .maintain/monitoring/alerting-rules/alerting-rules.yaml
```

Would it be a big performance impact to extend this to the whole `.maintain/monitoring` folder? There are not many pull requests targeting the folder per month. Should make this CI hook more future-proof in case we ever want to rename / restructure these files. 

```suggestion
      - .maintain/monitoring/
```

---

I don't mind tbh. A variable would have totally made sense, but changing a filename in that folder would also require a change in the `.gitlab-ci.yml` anyways. Please feel free to commit your changes and merge.

---

> require a change in the .gitlab-ci.yml anyways.

Good point. I still feel like this would be one thing less to watch out for.